### ISSUE TITLE
Redshift catalog

### ISSUE THREAD

**Issue by eyaltrabelsi** [Netflix/metacat#282]
Hi,
I see metacat support redshift connector, but still got some questions:
-  Do you have an example of catalog configuration for redshift? (I dont see in metacat/metacat-functional-tests/metacat-test-cluster/etc-metacat/catalog/)
-  How do I edit metacat.plugin.config.location to hold the catalog directory ? as environment variable? its not very clear in the README

**Comment by ajoymajumdar** [comment]
It should be similar to mysql or postgres.
Example:
`connector.name=redshift
metacat.schema.blacklist=information_schema
metacat.type=redshift

javax.jdo.option.name=redshift1
javax.jdo.option.url=jdbc:redshift://redshift-host:5439/dnflx?tcpKeepAlive=true&socketTimeout=100
javax.jdo.option.driverClassName=com.amazon.redshift.jdbc42.Driver
javax.jdo.option.username=user
javax.jdo.option.password=****
javax.jdo.option.maxActive=10
javax.jdo.option.maxIdle=5
javax.jdo.option.initialSize=0
javax.jdo.option.minIdle=2
javax.jdo.option.maxWait=5000
javax.jdo.option.minEvictableIdleTimeMillis=60000
javax.jdo.option.timeBetweenEvictionRunsMillis=10000
javax.jdo.option.testOnBorrow=true
javax.jdo.option.testWhileIdle=false
javax.jdo.option.testOnReturn=false
javax.jdo.option.removeAbandonedTimeout=120
javax.jdo.option.removeAbandoned=true
javax.jdo.option.logAbandoned=true
javax.jdo.option.validationQuery=SELECT 1
javax.jdo.option.jdbcInterceptors=org.apache.tomcat.jdbc.pool.interceptor.ConnectionState;org.apache.tomcat.jdbc.pool.interceptor.StatementFinalizer;org.apache.tomcat.jdbc.pool.interceptor.SlowQueryReportJmx(threshold=10000);com.netflix.metacat.common.server.util.PoolStatsInterceptor(name=redshift1)
javax.jdo.option.defaultTransactionIsolation=READ_COMMITTED
javax.jdo.option.defaultAutoCommit=true`

metacat.plugin.config.location should be passed as java options as in https://github.com/Netflix/metacat/blob/master/metacat-functional-tests/metacat-test-cluster/docker-compose.yml
`-Dmetacat.plugin.config.location=/etc/metacat/catalog` or in application.yml as 
`metacat
    plugin:
      config:
        location: /etc/metacat/catalog/testload`

**Comment by hautelooknaveen** [comment]
Hi @ajoymajumdar  should the catalog entry go in as a separate file like 'redshift.properties'. Also, should it be placed under "/etc/catalog" or "<METACAT_ROOT>/metacat-main/src/main/resources/etc/catalog" ? 

Currently, the java option "metacat.plugin.config.location" (_as set in application.yml)_ points to /etc/catalog   ... if so, can I just place redshift.properties under  the directory "/etc/catalog"?

**Comment by ajoymajumdar** [comment]
Yes, every catalog configuration needs to be  a separate file. By default, the catalog dir is /etc/catalog. So you can place it under /etc/catalog.

**Comment by hautelooknaveen** [comment]
Thanks @ajoymajumdar . Will give it a try.

**Comment by RoyalFeng** [comment]
hi,@ajoymajumdar ,If I want to use metacat connect to hive, how can i define the usermetadata.properties in /etc/metacat directory? can you give me the sample,thank you very much.

**Comment by ajoymajumdar** [comment]
@RoyalFeng usermetadata.properties is used to configure your metadata store where business metadata(definition and data metadata) about your schema is stored. To connect to your hive metastore, you can configure a hive connector catalog. Ex: https://github.com/Netflix/metacat/blob/master/metacat-functional-tests/metacat-test-cluster/etc-metacat/catalog/embedded-fast-hive-metastore.properties
